pkg/env/dev.go

```diff
@@ -87,7 +87,19 @@ func newDev(ctx context.Context, log *logrus.Entry, instancemetadata instancemet
 		roleassignments: authorization.NewRoleAssignmentsClient(instancemetadata.SubscriptionID(), armAuthorizer),
 	}
 
-	d.prod, err = newProd(ctx, log, instancemetadata)
+	config := auth.NewClientCredentialsConfig(os.Getenv("AZURE_RP_CLIENT_ID"), os.Getenv("AZURE_RP_CLIENT_SECRET"), os.Getenv("AZURE_TENANT_ID"))
```

Comment:
sorry, please add "AZURE_RP_CLIENT_ID" and "AZURE_RP_CLIENT_SECRET" to the list of checks at the top of the function.